basic/05-drive-your-computer-1.md

```diff
@@ -677,12 +677,12 @@ alias c++='c++-14'
 由于各类网络资源都会使用压缩文件的方式进行发布，寻找一款好用且**不流氓**的压缩软件很有必要。下面推荐一些常用的压缩软件：
 
 - Windows：
-  - `7-zip`或者其变种`7-zip-zstd`：老牌压缩软件，能满足大部分需求。官方网站：https://www.7-zip.org/download.html；`7-zip-zstd`官方网站：[GitHub - mcmilk/7-Zip-zstd: 7-Zip with support for Brotli, Fast-LZMA2, Lizard, LZ4, LZ5 and Zstandard](https://github.com/mcmilk/7-Zip-zstd)
-  - 老版`Bandizip`：请不要升级。新版本有广告。[Bandizip - 下载 Bandizip 6.x (bandisoft.com)](https://www.bandisoft.com/bandizip/old/6/)
+  - `7-zip`或者其变种`7-zip-zstd`：老牌压缩软件，能满足大部分需求。[`7-zip`官方网站](https://www.7-zip.org/download.html)；[`7-zip-zstd`官方网站](https://github.com/mcmilk/7-Zip-zstd)
+  - 老版 `Bandizip`：请不要升级。新版本有广告。[Bandizip - 下载 Bandizip 6.x (bandisoft.com)](https://www.bandisoft.com/bandizip/old/6/)
   - `WinRAR`：老牌压缩软件。优点是可以生成 RAR 文件，且功能较全、界面不错。**请注意**：若要使用此软件，有如下注意事项：
     - 必须从英文官网下载：[WinRAR download free and support: WinRAR (win-rar.com)](https://www.win-rar.com/start.html?&L=0)中文版无论是否购买正版都有广告，而英文版无论是否购买正版都没有广告，也不会有弹窗。30 天试用期不会减少。
     - **版本号必须大于 7.0.0！！！**这是因为，以前的版本有一个重要**高危漏洞**，可以无需以`.exe`后缀存放文件，执行恶意代码。
-    - 可以参考如下网站进行**个人学习**：[GitHub - bitcookies/winrar-keygen: Principle of WinRAR key generation.](https://github.com/bitcookies/winrar-keygen?tab=readme-ov-file)请参考此网站的 README 的**6.3**节。
+    - 可以参考如下网站进行**个人学习**：[GitHub - bitcookies/winrar-keygen](https://github.com/bitcookies/winrar-keygen?tab=readme-ov-file)请参考此网站的 README 的**6.3**节。
```

Comment:
Suggest removal of this line. We are publishing our notes and cannot contain cracking method or pirate software

Reply:
done